src/renderer/renderScene.ts

```diff
@@ -102,6 +104,17 @@ export function renderScene(
     return { atLeastOneVisibleElement: false };
   }
 
+  if (appState.wysiwygElement && appState.wysiwygElement.changeStyle) {
+    if (isTextElement(appState.editingElement)) {
+      appState.wysiwygElement.changeStyle({
+        font: (appState.editingElement as ExcalidrawTextElement).font,
+        textAlign: (appState.editingElement as ExcalidrawTextElement).textAlign,
+        color: (appState.editingElement as ExcalidrawTextElement).strokeColor,
+        opacity: (appState.editingElement as ExcalidrawTextElement).opacity,
```

Comment:
can we avoid casting here somehow?

Reply:
That's what I suggested as edits. Where did that go? :)

Reply:
Since we're already using a type guard in the condition, all those assertions can now be simply dropped.